src/app/components/TranscriptExperiment/MediaIndicator/index.test.tsx

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable prettier/prettier */
+import React from 'react';
+import { render } from '../../react-testing-library-with-providers';
+
+import MediaIcon from '.';
+
+describe('MediaIcon', () => {
+  afterEach(() => {
+    jest.resetAllMocks();
+  });
+
+  it(`should render a mini play button`, () => {
+    const { container } = render(
+      <MediaIcon
+        title="My Video"
+        duration="2:30"
+        durationSpoken="2 minutes 30 seconds"
+        datetime="PT2M30S"
+        guidanceMessage="Guidance: May contain strong language that may offend."
+      />,
+      {
+        service: 'news',
```

Comment:
Can we use a service other than news as our example - mundo/ pidgin? (In fact news is already the default, so let's override it with one of our language services)